components/pages/workPageLayout/WorkPageButtons.tsx

```diff
@@ -1,6 +1,5 @@
-import React, { useMemo, useState } from "react"
+import React, { useMemo } from "react"
```

Comment:
I think the`WorkPageButtons` component is too big and its content should be divided into several components. But we already talked about that, so there is no need to dwell into that.